src/data/defaults.txt

```diff
@@ -159,6 +159,7 @@ global	useLastUserAgent	false
 global	useSystemTrayIcon	false
 global	useTabbedChatFrame	true
 global	useToolbars	true
+global	useCachedVolcanoMaps	true
```

Comment:
Is this worth letting be configurable? When would a user want to turn this off?

Reply:
Oh hmm auto-merge sure interferes with my approach of approving with optional comments!

Reply:
My tests that demonstrate that we can derive new map sequences turn it off.
The user will not want to turn it off. It need not be configurable.

Sorry about automerge.

Reply:
It's okay, I should have checked myself.